tcc-transaction-unit-test/src/test/java/org/mengyun/tcctransaction/repository/helper/RedisHelperTest.java

```diff
@@ -0,0 +1,71 @@
+package org.mengyun.tcctransaction.repository.helper;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.powermock.api.mockito.PowerMockito;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+import java.util.Arrays;
+
+public class RedisHelperTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void testGetRedisKey() {
+    Assert.assertArrayEquals(new byte[] {(byte)58, (byte)58},
+        RedisHelper.getRedisKey("", ":", ""));
+
+    Assert.assertEquals(Arrays.toString(new Byte[] {65, 71, 71, 103,
```

Comment:
Can you not remove the `toString` and use an assertArrayEquals? It may be beacuse you are using a `Byte` with a capital `B`?

Reply:
done